frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-as-code.feature

```diff
@@ -0,0 +1,83 @@
+                  | test-repo       |
+        
+        @smoke
+        Scenario Outline: Repositories page display on newly created repository: P-11-TC03
+            Given repository "<repository_name>" is present on Repositories page
+             When user searches repositoy "<repository_name>" in repositories page
+             Then repositories table displayed with column names Name, Event type, Last run, Task status, Last run status, Last run time, Last run duration
+              And clolumn Name display with value "<repsitory_name>"
+              And columnsLast run, Task status, Last run status, Last run time, Last run duration with values display "-"
+              And Create button is enabled
+              And kebab menu button is displayed
+              
+        Examples:
+                  | repository_name |
+                  | test-repo       |
+        
+        @smoke
+        Scenario: Kebab menu options of newly created repository in Repositories page: P-11-TC04
+            Given repository "test-repository" is present on Repositories page
+             When user searches repository "test-repository" in repsitories page
+              And user clicks on kebab menu of the repository "test-repository"
+             Then kebab menu displays with options Edit labels, Edit annotations, Edit repository, Delete repository
+        
+        @smoke
+        Scenario: Actions menu of newly created reposiotry in Repository details page: P-11-TC05
+            Given useris at repositories details page with newly created repository "test-repo"
+             When user clicks Actions menu in the repository details page
+             Then Actions menu display with options Edit labels, Edit annotations, Edit repository, Delete repository
+        
+        @regression
+        Scenario Outline: Delete the repository from the Repository details page: P-11-TC06
+            Given repository "<repository_name>" is present on Repositories page
+             When user searches repository "<repository_name>" in repositories page
+              And user clicks repository "<repository_name>" from searched results on Repositories page
+              And user selects option "Delete Repository" from Actions menu drop down
+              And user clicks Delete button on Delete Repository modal
+             Then user will be redirected to Repositories page
+              And "<repository_name>" is not displayed on Pipelines page
+
+        Examples:
+                  | repository_name |
+                  | test-repo       |
+
+        @regression
+        Scenario: Delete the Repository from repositories page: P-11-TC07
+            Given repository "test-repo" is present on Repositories page
+             When user selects "Delete Repository" from the kebab menu for "test-repo"
+              And user clicks Delete button on Delete Reporsitory modal
+             Then user will be redirected to Repositories page
```

Comment:
Same action is performed in above scenario, So IMO this scenario is not required. Instead we can edit the repository

Reply:
The only way we can edit the repository is by editing yaml content

Reply:
That is fine, please add it

Reply:
Added